src/PanoImageRenderer/renderer/SphereRenderer.js

```diff
@@ -50,6 +50,11 @@ export default class SphereRenderer extends Renderer {
 			}`;
 	}
 
+	static updateTexture(gl, texture, image) {
+		// Draw first frame
+		this.texImage2D(gl, image);
```

Comment:
I think it's better to remove texImage2D method and replace this function with following

```js
static updateTexture(gl, image) {
  gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE, image);
}
```